tests/test_e2e_01_kytos_startup.py

```diff
@@ -111,6 +119,7 @@ def test_start_kytos_api_core(self):
     # test auth api
     # TODO
 
+    @pytest.mark.skipif(os.path.exists('/var/log/syslog'), reason="/var/log/syslog does not exist")
```

Comment:
Should this be `not os.path.exists('/var/log/syslog')`? As it is, you're sking if the file exist, and if it doesn't then it'll try to read something that doesn't exist, so `open` would crash.